CONTRIBUTING.md

```diff
@@ -59,5 +59,5 @@ Please follow these steps to make your first PR!
     ```
     $ git push origin BRANCH_NAME
     ```
-9.  Create a pull request by going to your repository fork on GitHub and clicking "Compare & pull request)
+9. Create a pull request by going to your repository fork on GitHub and clicking "Compare & pull request")
```

Comment:
LIne 62, please change ')' to a period. I think this will be the last thing you have to change. Thanks! : )

Reply:
Geez I should've caught that earlier tonight, guess I'm tired. Fixed now though :)